appserver/common/glassfish-naming/src/main/java/com/sun/enterprise/naming/impl/GlassfishNamingManagerImpl.java

```diff
@@ -684,7 +684,7 @@ private ArrayList<SimpleJndiName> listNames(SimpleJndiName name) throws NamingEx
     private String getComponentId() throws NamingException {
         final ComponentInvocation ci;
```

Comment:
At some point (maybe not now) make these things "componentInvocation" instead of "ci", etc,

Reply:
I agree, but then I would have to go through the whole module as this is "a convention" here if I remember right.
But I will take a look once again in the morning, I already made a typo.